content/master/concepts/compositions.md

```diff
@@ -1315,8 +1315,10 @@ inside a Composition. This verifies that the resource `apiVersion` and `kinds`
 are valid. 
 <!-- vale write-good.TooWordy = YES -->
 
-Enable "schema validation" with the
-`--enable-composition-webhook-schema-validation` flag on the Crossplane pod.
+Crossplane performs compositions "schema validation" by default. However, if you
+prefer to bypass this functionality, you can set the
+`--enable-composition-webhook-schema-validation=false` flag on the Crossplane
+pod.
```

Comment:
```suggestion
Crossplane performs compositions "schema validation" by default.  
Disable validation with the 
`--enable-composition-webhook-schema-validation=false` flag on the Crossplane
pod.
```

Reply:
Update to pass Vale

Reply:
changed, have a look at it, I managed to make vale happy 🎉